src/common/packet/packet_type.h

```diff
@@ -69,7 +69,7 @@ typedef enum PacketType {
 	CB_REQ_GET_POSTBOX_ITEM = 91, // size = 30
 	BC_WAIT_QUEUE_ORDER = 92, // size = 10
 	CB_CANCEL_SERVER_WAIT_QUEUE = 93, // size = 10
-	CS_LOGIN = 2901, // size = 64
+	CS_LOGIN = 2901, // size = 68

```

Comment:
The old value has been extracted from the client - There might be something wrong with this new value